querydsl-sql/src/test/java/com/querydsl/sql/SelectBase.java

```diff
@@ -232,6 +232,13 @@ public void Boolean_Any() {
     }
 
     @Test
+    public void Case() {
```

Comment:
I see you only updated `JPQLTemplates` so I'm guessing the issue only existed in JPA and you're just adding a test for the SQL suite.